apps/easypid/src/features/didcomm/DidCommNotificationScreen.tsx

```diff
@@ -54,8 +55,8 @@ export function DidCommNotificationScreen() {
   const onComplete = () => handleNavigation('replace')
 
   const onConnectionAccept = async () => {
-    const result = await acceptConnection()
-    if (result.success) {
```

Comment:
I'm kinda a fan of the `success` and then having data or error, since:
- try/catch is a bit funky in react
- it allows you to provide more specific errors metadata (like for parse invitation what the error type is). If we want to do that with error you need more documentation on the error types, while with typescript the return type can already include this. 
- You need to either check if error is instance of error, or cast as an error. Bot are suboptimal

So if we are changing this model we should also think about how the user of the SDK will deal with errors.